docs/user-manual.md

```diff
@@ -30,6 +30,17 @@ To stop the pie, follow these steps:
    $ CRONJOBS_LIST=$(kubectl -n ${NAMESPACE} get cronjobs --no-headers=true | awk '{print $1}')
    $ [ -n "${CRONJOBS_LIST}" ] && kubectl -n ${NAMESPACE} delete cronjobs ${CRONJOBS_LIST}
    ```
+4. Remove the pie's finalizers for the Pods.
+   ```console
+   $ PODs=$(kubectl get pod -n ${NAMESPACE} -o json | jq -r '.items[] | .metadata.name')
```

Comment:
Could you add a label filter to avoid removing the finalizer from the pods that are not related to pie?

Reply:
Unfortunately, there seem to be no meaningful labels here. Let me think about how we can work around this.

Reply:
I updated my PR using `grep -E` instead of label filters ([the changes](https://github.com/topolvm/pie/compare/f21bbedd9a43503ee9eda8507a1dbc575e179c50..1cb76dc95662cf74e29fd064f8c6fd32b544cea2)). Could you review this again?